docs/collector.mssql.md

```diff
@@ -208,7 +208,7 @@ Name | Description | Type | Labels
 `windows_mssql_memmgr_allocated_lock_blocks` | Specifies the current number of allocated lock blocks. At server startup, the number of allocated lock blocks plus the number of allocated lock owner blocks depends on the SQL Server Locks configuration option. If more lock blocks are needed, the value increases | counter | `mssql_instance`
 `windows_mssql_memmgr_lock_memory_bytes` | Specifies the total amount of dynamic memory the server is using for locks | counter | `mssql_instance`
 `windows_mssql_memmgr_lock_owner_blocks` | Specifies the current number of allocated lock owner blocks. At server startup, the number of allocated lock owner blocks and the number of allocated lock blocks depend on the SQL Server Locks configuration option. If more lock owner blocks are needed, the value increases dynamically | counter | `mssql_instance`
-`windows_mssql_memmgr_allocated_lock_owner_blocks` | _Not yet documented_ | counter | `mssql_instance`
+`windows_mssql_memmgr_allocated_lock_owner_blocks` | The current number of allocated lock owner blocks | counter | `mssql_instance`
```

Comment:
The description for this metric indicates it would be a gauge due to the "current" keyword. Does that appear correct to you?